.travis.yml

```diff
@@ -7,6 +7,8 @@ addons:
     - github.com 
 node_js:
   - 8
+  - 10
```

Comment:
Should we add 9 as well ?

Reply:
I don't think so, odd versions are only supported for a short time and I don't think we need to support node versions that aren't supported for cli.  In the future I can see doing the leading version like 13 for early adopters and it helps us make sure we are in a good position for the next even/lts release.

Thoughts?

@junedkazi @bookernath @bc-williamkwon 

https://nodejs.org/en/about/releases/